app/poros/github_results.rb

```diff
@@ -0,0 +1,12 @@
+class GithubResults
+  def repos
+    body = GithubService.new.repos_body
+    body.map do |repo_data|
+      Repo.new(repo_data)
+    end
+  end
+
+  def repos_limit(number)
+    repos[0..number - 1]
+  end
```

Comment:
only return 5 repo objects

Reply:
this is so much simpler than what I thought it would be. Awesome!